lib/spree_multi_currency/engine.rb

```diff
@@ -1,6 +1,9 @@
 module SpreeMultiCurrency
   class Engine < Rails::Engine
     require 'spree/core'
+    require 'money'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.